be/src/exec/pipeline/sink/iceberg_table_sink_operator.h

```diff
@@ -31,14 +32,15 @@ class IcebergTableSinkOperator final : public Operator {
 public:
     IcebergTableSinkOperator(OperatorFactory* factory, int32_t id, int32_t plan_node_id, int32_t driver_sequence,
                              std::string location, std::string file_format, TCompressionType::type compression_codec,
-                             IcebergTableDescriptor* iceberg_table, FragmentContext* fragment_ctx,
-                             const std::shared_ptr<::parquet::schema::GroupNode>& schema,
+                             const TCloudConfiguration& cloud_conf, IcebergTableDescriptor* iceberg_table,
+                             FragmentContext* fragment_ctx, const std::shared_ptr<::parquet::schema::GroupNode>& schema,
                              const std::vector<ExprContext*>& output_expr_ctxs,
                              const vector<ExprContext*>& partition_output_expr)
             : Operator(factory, id, "iceberg_table_sink", plan_node_id, driver_sequence),
               _location(std::move(location)),
               _file_format(std::move(file_format)),
               _compression_codec(std::move(compression_codec)),
+              _cloud_conf(cloud_conf),
```

Comment:
check is null

Reply:
added null check for all optional fields